category.html

```diff
@@ -66,10 +66,13 @@ <h1>GAMERS</h1>
                 <a class="nav-link" href="cart.html">Cart </a>
               </li>
               <li class="nav-item">
```

Comment:
The cards should be clickable to lead to the category page!!!

Reply:
done